### ISSUE TITLE
[Project question] Mouse coordinates in button-press-event

### ISSUE THREAD

**Issue by Hultan** [gotk3/gotk3#772]
Hi,

I have an Application Window that contains a gtk.Fixed, which in turn contains a gtk.TextView that does not cover the entire gtk.Fixed.

    -------------------------------------------------------
    | gtk.Fixed                                           |
    |                                                     |
    |               ----------------------------          |
    |               | gtk.TextView             |          |
    |      1        |                          |          |
    |               |              2           |          |
    |               ----------------------------          |  
    |                                                     |
    -------------------------------------------------------

I have connected the button-press-event signal, and I get the mouse coordinates using the following code:

```
func (v *viewedPage) clicked(fixed *gtk.Fixed, e *gdk.Event) bool {
	eventButton := gdk.EventButtonNewFromEvent(e)
	if eventButton.Button() != gdk.BUTTON_PRIMARY {
		return false
	}

	eventMotion := gdk.EventMotionNewFromEvent(e)
	x, y := eventMotion.MotionVal()

	fmt.Println(x,y)
}
```
Here is the problem that I am facing: 

If I click where the digit 1 is, inside the gtk.Fixed-area (but outside of the gtk.TextView!), I get coordinates relative the upper left corner of the gtk.Fixed which is exactly what I want. 

However, if I click where the digit 2 is, inside the gtk.TextView and the gtk.Fixed, I get coordinates relative the gtk.TextView, which is NOT what I want. The coordinates starts at 0,0 again in the upper left coordinates of the gtk.TextView.

I always want the gtk.Fixed coordinates, and given that the signal is connected to the gtk.Fixed, that is what I would think that you should get, but no?!?

Why do I get coordinates from two different coordinate systems? How can I fix this?

[Edit] I have tried the TranslateCoordinates-function, but I could not get that to work either.

**Comment by Hultan** [comment]
Here is a code example : https://github.com/Hultan/testCoordinates

**Comment by Hultan** [comment]
A tiny little bit closer : If I add the line textView.SetHasWindow(false) the coordinates reported in the event seems to be correct, but it's placement is wrong (not 100,100 but rather ~200,200) and it does not show the text.

If i set textView.SetHasWindow(true), the coordinates reported in event is wrong, but placement is correct and the text is showing.

Is this a bug? Or have I misunderstood something? I have updated the code on github...

**Comment by Hultan** [comment]
Since I didn't get any good answers here, I decided to do it in a very convoluted way. I imported RobotGO (https://github.com/go-vgo/robotgo) to get the screen coordinates, and the subtracted the form position, and I got some coordinates I could work with.

There must be an easier way than this, but hey, it works for now. If this application that I am working on ever goes public, I hopefully have found a better way to do this.
